src/db/spanner/schema.ddl

```diff
@@ -11,25 +11,28 @@
 --       ALSO, CONSOLE WANTS ONE SPACE BETWEEN DDL COMMANDS
 
 CREATE TABLE user_collections (
-  fxa_uid STRING(MAX)  NOT NULL,
-  fxa_kid STRING(MAX)  NOT NULL,
-  collection_id INT64  NOT NULL,
-  modified TIMESTAMP   NOT NULL,
+  fxa_uid STRING(MAX) NOT NULL,
```

Comment:
The changes to the schema reflect the most up-to-date schema pulled from the GCP console by @jrconlin